packages/formation-react/src/components/DropDownPanel/DropDownPanel.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import DropDownPanel from './DropDownPanel';
+
+export default {
+  title: 'Library/DropDownPanel',
+  component: DropDownPanel,
+};
+
+const Template = (args) => {
+  const [isOpen, setOpen] = useState(args.isOpen);
+
+  return (
+    <div style={{ backgroundColor: '#112e51' }}>
+      <DropDownPanel
+        {...args}
+        isOpen={isOpen}
+        clickHandler={() => setOpen(!isOpen)}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce
+        ullamcorper at eros eu suscipit. Ut imperdiet libero et luctus pretium.
```

Comment:
I saw the story and looked at the component code locally before I looked here, so I was wondering where this came from. Unfortunately, according to [the docs](https://storybook.js.org/docs/react/workflows/stories-for-multiple-components#using-children-as-an-arg) it looks like we can't have this be dynamic:

>As things stand (we hope to improve this soon) you cannot edit children in a control yet.

Reply:
:sigh: Yup.